R/summaries.R

```diff
@@ -13,7 +13,10 @@
 #'   median or prevalence. Default is 'mean'.
 #'
 #' @param abund_values A \code{character} value to select an
-#'   \code{\link[SummarizedExperiment:SummarizedExperiment-class]{assayNames}}
+#'   \code{\link[SummarizedExperiment:SummarizedExperiment-class]{assayNames}} 
+#'   
+#' @param remove_na For \code{getTopTaxa} logical argument for calculation method 
```

Comment:
I suggest to use the argument name `na.rm` which is used in many different functions and packages. It would be more standard.

Reply:
OK will make the change 😀